% This is the Reed College LaTeX thesis template. Most of the work 
% for the document class was done by Sam Noble (SN), as well as this
% template. Later comments etc. by Ben Salzberg (BTS). Additional
% restructuring and APA support by Jess Youngberg (JY).
% Your comments and suggestions are more than welcome; please email
% them to cus@reed.edu
%
% See http://web.reed.edu/cis/help/latex.html for help. There are a 
% great bunch of help pages there, with notes on
% getting started, bibtex, etc. Go there and read it if you're not
% already familiar with LaTeX.
%
% Any line that starts with a percent symbol is a comment. 
% They won't show up in the document, and are useful for notes 
% to yourself and explaining commands. 
% Commenting also removes a line from the document; 
% very handy for troubleshooting problems. -BTS

% As far as I know, this follows the requirements laid out in 
% the 2002-2003 Senior Handbook. Ask a librarian to check the 
% document before binding. -SN

%%
%% Preamble
%%
% \documentclass{<something>} must begin each LaTeX document
\documentclass[12pt,twoside]{reedthesis}
% Packages are extensions to the basic LaTeX functions. Whatever you
% want to typeset, there is probably a package out there for it.
% Chemistry (chemtex), screenplays, you name it.
% Check out CTAN to see: http://www.ctan.org/
%%
\usepackage{graphicx,latexsym} 
\usepackage{amssymb,amsthm,amsmath}
\usepackage{longtable,booktabs,setspace} 
\usepackage{chemarr} %% Useful for one reaction arrow, useless if you're not a chem major
\usepackage[hyphens]{url}
\usepackage{rotating}
\usepackage{natbib}
\usepackage{tabularx}
\usepackage[
singlelinecheck=false % <-- important
]{caption}
\usepackage{float}

% Comment out the natbib line above and uncomment the following two lines to use the new 
% biblatex-chicago style, for Chicago A. Also make some changes at the end where the 
% bibliography is included. 
%\usepackage{biblatex-chicago}
%\bibliography{thesis}

% \usepackage{times} % other fonts are available like times, bookman, charter, palatino

\title{Determination of BosR DNA Binding Sequence Using CASTing Method}
\author{Sofia-Mari Claesson}
% The month and year that you submit your FINAL draft TO THE LIBRARY (May or December)
\date{May 2016}
\thedivisionof{The Established Interdisciplinary Committee for Biochemistry and Molecular Biology}
\advisor{Arthur Glasfeld}
%If you have two advisors for some reason, you can use the following
%\altadvisor{Your Other Advisor}
%%% Remember to use the correct department!
\approvedforthe{Committee} 
% if you're writing a thesis in an interdisciplinary major,
% uncomment the line below and change the text as appropriate.
% check the Senior Handbook if unsure.
%\thedivisionof{The Established Interdisciplinary Committee for}
% if you want the approval page to say "Approved for the Committee",
% uncomment the next line
%\approvedforthe{Committee}

\setlength{\parskip}{0pt}
%%
%% End Preamble
%%
%% The fun begins:
\begin{document}

  \maketitle
  \frontmatter % this stuff will be roman-numbered
  \pagestyle{empty} % this removes page numbers from the frontmatter
  
  \newcommand{\sofiachapter}[1]{
  	\chapter*{#1}
  	\addcontentsline{toc}{chapter}{#1}
  	\chaptermark{#1}
  	\markboth{#1}{#1}
  }
  
    \setcounter{secnumdepth}{0}

% Acknowledgements (Acceptable American spelling) are optional
% So are Acknowledgments (proper English spelling)
    \chapter*{Acknowledgements}
	%I want to thank a few people.

% The preface is optional
% To remove it, comment it out or delete it.
    \chapter*{Preface}
	%This is an example of a thesis setup to use the reed thesis document class.
	
	

    \chapter*{List of Abbreviations}
		%You can always change the way your abbreviations are formatted. Play around with it yourself, use tables, or come to CUS if you'd like to change the way it looks. You can also completely remove this chapter if you have no need for a list of abbreviations. Here is an example of what this could look like:

	\begin{table}[h]
	\centering % You could remove this to move table to the left
	\begin{tabular}{ll}
		\textbf{Bb}  	&  \textit{Borrelia burgdorferi} \\
		\textbf{rBosR}  	&  Recombinant BosR\\
		%\textbf{CDC}  	&  Center for Disease Control \\
		%\textbf{CIA}  	&  Central Intelligence Agency\\
		%\textbf{CLBR} 	&  Center for Life Beyond Reed\\
		%\textbf{CUS}  	&  Computer User Services\\
		%\textbf{FBI}  	&  Federal Bureau of Investigation\\
		%\textbf{NBC}  	&  National Broadcasting Corporation\\
	\end{tabular}
	\end{table}
	

    \tableofcontents
% if you want a list of tables, optional
    \listoftables
% if you want a list of figures, also optional
    \listoffigures

% The abstract is not required if you're writing a creative thesis (but aren't they all?)
% If your abstract is longer than a page, there may be a formatting issue.
    \chapter*{Abstract}
	%The preface pretty much says it all.
	
	\chapter*{Dedication}
	%You can have a dedication here if you wish.

  \mainmatter % here the regular arabic numbering starts
  \pagestyle{fancyplain} % turns page numbering back on

%The \introduction command is provided as a convenience.
%if you want special chapter formatting, you'll probably want to avoid using it altogether

   \sofiachapter{Introduction}
         
	% The three lines above are to make sure that the headers are right, that the intro gets included in the table of contents, and that it doesn't get numbered 1 so that chapter one is 1.

% Double spacing: if you want to double space, or one and a half 
% space, uncomment one of the following lines. You can go back to 
% single spacing with the \singlespacing command.
% \onehalfspacing
\doublespacing
	
	%Welcome to the \LaTeX\ thesis template. If you've never used \TeX\ or \LaTeX\ before, you'll have an initial learning period to go through, but the results of a nicely formatted thesis are worth it for more than the aesthetic benefit: markup like \LaTeX\ is more consistent than the output of a word processor, much less prone to corruption or crashing and the resulting file is smaller than a Word file. While you may have never had problems using Word in the past, your thesis is going to be about twice as large and complex as anything you've written before, taxing Word's capabilities. If you're still on the fence about  using \LaTeX, read the Introduction to LaTeX on the CUS site as well as skim the following template and give it a few weeks. Pretty soon all the markup gibberish will become second nature.
%put words here that should never be hyphenated in the document	
\hyphenation{EMSAs}
\hyphenation{rpoS}
\hyphenation{BosR}
\hyphenation{bosR}
\hyphenation{rpoN}

\section{What is \textit{Borrelia burgdorferi}?}
\newcommand{\FAstdfit}{r = $\frac{\Delta r \cdot P}{K_{d} + P }$ + $r_{min}$}
\newcommand{\micro}{$\mu$}


	\textit{Borrelia burgdorferi} is a spiral, rod-shaped bacterium, and the etiological agent of Lyme disease. Since the 1970s, Lyme disease has become a rapidly emerging infectious disease in the United States. Over 90\% of vector-borne illnesses in the US are attributed to Lyme disease and it was ranked as the fifth most common Nationally Notifiable disease in 2014 (Radolf et al., 2012 (ref 8) and CDC 2015). While around 30,000 cases of Lyme disease are reported annually in the US, the Center for Disease Control estimates that the actual  number of cases is approximately 300,000 (CDC 2015). Infected ticks transmit the \textit{Borrelia burgdorferi} when feeding, leading to human infection. Early onset symptoms of Lyme disease include flu-like symptoms, such as fever, fatigue, and chills, as well as muscle or joint pain. The most tell-tale symptom of Lyme disease is an erythema migrans rash, which is identifiable by its "bulls-eye" shape. The rash, however, does not present in every case. The absence of this symptom decreases the likelihood of early diagnosis. Late stage symptoms can be more severe, including arthritis, joint swelling, heart palpitations, dizziness, shortness of breath, brain and spinal cord inflammation, and nerve pain. Neurological symptoms such as facial paralysis and short-term memory loss may also occur (CDC 2015). Often ticks can transmit other co-infections, such as Babesiosis and Ehrlichiosis, further complicating diagnosis and treatment (Krause et al., 2002). Left untreated, a broad range of symptoms are associated with Lyme disease (CDC 2015). 
	
	Lyme disease occurrence in the US is not geographically uniform. Lyme disease is most commonly found in the northeast and the upper midwest (Figure 1).  Vector distribution and presence of the bacteria influence the distribution of the disease. (CDC, 2015).
		 	 	\begin{figure}[h]
		 	 		% the options are h = here, t = top, b = bottom, p = page of figures.
		 	 		% you can add an exclamation mark to make it try harder, and multiple
		 	 		% options if you have an order of preference, e.g.
		 	 		% \begin{figure}[h!tbp]
		 	 		
		 	 		\centering
		 	 		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
		 	 		\includegraphics{Figures_Images/lyme_prevalence0711}
		 	 		\caption[Geographic Distribution and Prevalence of Lyme Disease in the US]{Map of Lyme disease occurrence in the US between 2007-2011 showing geographic distribution and prevalence on the county level. Data used were counts of Lyme disease cases in each US county provided by the CDC. Prevalence was calculated as the number of confirmed cases in each county during 2007-2011 divided by the county population, as given by the 2000 census. Results were log transformed to better visualize relative differences. Bright green corresponds to high prevalence, while white, corresponds to zero.}
		 	 		% square brackets here correspond to what is written in the list of figures
		 	 		\label{Lymemap}
		 	 		% this is label for the figure
		 	 	\end{figure}
		 	 	%cite ggplot packages used to create image
 	 	
	
	 Species from the genus \textit{Borrelia} are divided into two groups, responsible either for Lyme disease or relapsing fever. Those species associated with Lyme disease have been categorized as belonging to the \textit{Borrelia burgdorferi sensu lato} complex. Within this complex three strains have been identified as the primary causes of Lyme disease (Radolf et al., 2012 (cites Borrelia: Molecular bio...from Samuels and Radolf)).\textit{B. burgdorferi sensu stricto} is the species that is mostly responsible for causing Lyme disease in the US, while \textit{Borrelia garinii} and \textit{Borrelia afzelii} cause the disease in Asia (Radolf et al.,2012 \& Masuzawa, 2004). All three strains are present in Europe (Radolf et al., 2012 (ref 7)). The bacteria are transmitted to mammalian hosts through an arthropod vector, mainly various species of the genus \textit{Ixodes}, hard-bodied ticks (Radolf et al., 2012 and Burgdorfer et al., 1982). \textit{Ixodes scapularis} and \textit{Ixodes pacificus}, the blacklegged tick and western blacklegged tick, have been identified as species primarily responsible for transmission of the bacteria in North America (Radolf et al., 2012 (ref 7)). \textit{Ixodes ricinus} and \textit{Ixodes persulcatus} are the primary species responsible for Lyme disease in Europe and Asia, respectively (Dizij \& Kurtenbach, 1995; Chengxu et al., 1988). 
	
	
	The \textit{Borrelia burgdorferi} bacteria, therefore, have the challenge of adapting to two very different environments, the tick and the mammalian host. The life cycle of the bacterium starts when the tick larva feed on small mammals such as birds, mice, and squirrels, which are their first hosts. Small birds, mice, and squirrels have \textit{Borrelia burgdorferi} in their blood, and it is transmitted to the tick during feeding. The tick larva fall off their first host when they are done feeding and have possibly acquired \textit{Borrelia burgdorferi}. The tick then matures to a nymph and feeds on a second host. The same types of small birds, mice, and squirrels that the tick fed on in the larval stage serve as second hosts. During this point the tick can transmit the bacteria to a small mammal that was previously uninfected, thereby perpetuating the cycle. The tick can also feed on incidental hosts that do not perpetuate the cycle, such as humans or dogs, and transmit \textit{Borrelia burgdorferi} to them, resulting in infection that causes Lyme disease. The adult tick could also feed on a third host, such as a deer, on which the tick can lay eggs and start the cycle all over again  (Radolf et al., 2012 (ref 7)). 
	
	Bites from nymphal ticks are often the cause of human infection. It usually requires 36-48 hours of attachment for the tick to transmit the bacteria. Since nymphal ticks are less than 2mm or about the size of a poppy seed, it can be difficult to find them on the body, allowing time for transmission of \textit{Borrelia burgdorferi}. In addition, the ticks' saliva, which transmits the bacteria, contains anesthetic properties so that the host is unable to feel the bite. Nymphs are most common when they are feeding in spring and summer, while adult ticks are more common in fall (CDC, 2015). Climate warming is expected to expand the distribution of ticks that transmit \textit{Borrelia burgdorferi}. Warmer temperatures could spread the disease into areas in which it previously was not a concern and could increase risk of Lyme disease in areas in which it is already endemic (Ogden et al., 2014). In northwestern California, nymph \textit{Ixodes pacificus} are active from January through October and adult ticks are active from late October to June. Both nymphs and adults can potentially transmit \textit{Borrelia burgdorferi}, posing a year-round potential for contraction of Lyme disease in California. This finding revealed an expanded time frame for tick activity (Salkeld et al., 2014). The active seasons of ticks coincide with when people spend the most time outdoors, making preventative measures and awareness extremely important. 
	
	
	%add about ticks in spring in summer due to temperature and how warming climate could increase the season
	%add how ticks locate hosts and discharge anesthetic saliva when biting + how protein salp15 has been show to be immunosuppressive
	
	
	 \textit{Borrelia burgdorferi} has many unusual characteristics. It has a linear rather than circular chromosome and has a variety of circular and linear plasmids. The \textit{Borrelia burgdorferi} type strain B31, the first \textit{Borrelia} genome sequenced, has a 910 Kbp linear chromosome and 12 linear and nine circular plasmids, for a total of 610 Kbp of plasmid DNA (Fraser et al., 1997 \& Casjens et al., 2000). The plasmids contain a large number of non-functional pseudogenes, which is likely due to DNA rearrangement events in linear plasmids that produced duplications. More than 90\% of the plasmid genes are also not similar to those outside of the \textit{Borrelia} genus (Casjens et al., 2000). Genomic analysis of commonly used laboratory strains of \textit{Borrelia  burgdorferi}, including B31, shows variability in the identity of plasmids that are present. Linear plasmids appear to be especially variable. Plasmids lp54 and cp26, however, are quite stable (Casjens et al., 2012). These plasmids both contain genes that encode outer surface lipoproteins that are essential for the bacterium's survival in both the tick and mammal. Genes encoded by these plasmids can be differentially expressed to suit the environment of the bacteria. Plasmid lp54 encodes outer surface protein A and B (OspA and OspB). Plasmid cp26 encodes outer surface protein C (OspC), which is essential for mammalian infection (Casjens et al., 2012; Grimm eta la., 2004). OspA is not necessary for the bacteria to establish mammalian infection, but is needed for the bacteria to colonize and survive in the midgut of the tick (Yang et al., 2004). OspA protects the bacteria from host antibodies in the blood that ticks uptake during feeding (Battisti et al., 2008). Expression of OspA decreases as the bacteria move into the tick salivary gland out of the midgut, along with an increase in expression of OspC (Samuels, 2011).    (Casjens et al., 2012). In the early stages of infection, OspC expression is downregulated in the bacteria as a mechanism for immune system evasion (Tilly et al., 2006; Liang et al., 2002). Environmental cues, such as temperature and pH, signal when to express OspA and OspC. There is conflicting evidence in the literature as to whether the regulation of OspA and OspC is independent or correlated (Samuels, 2011). Silencing of the \textit{ospAB} operon results in increased production of OspC as well as other lipoproteins, suggesting that the regulation of OspA and OspC is not exclusively independent (He et al., 2008). Along with several other borrelial proteins,  regulation of OspAB and OspC is important for bacterial survival throughout the enzootic cycle. 
	 
	 In \textit{Escherichia coli} and other related bacteria, the alternative sigma factor RpoS functions as a global regulator of the general stress response by altering expression of dependent genes that have protective functions (Hengge-Aronis, 2002). In \textit{Borrelia burgdorferi} RpoS is not fundamental to the general stress response, but is involved in the activation of OspC and decorin-binding protein A (DbpA), another lipoprotein involved in mammalian infection (Caimano et al., 2004; H\"ubner et al., 2001). RpoS in \textit{Borrelia} is also necessary for the spirochete to move to the tick salivary gland to be transmitted to hosts (Samuels, 2011; Fisher et al., 2005). Analysis of the transcriptome suggests
	 that it is also directly or indirectly involved in the expression of over a hundred genes, many of which participate in maintenance during the enzootic cycle  (Samuels, 2011). 
	 
		In \textit{Borrelia burgdorferi} there are two pathways for RpoS transcription that lead to either a short or long transcript of mRNA. The short RpoS transcript is produced through an RpoN dependent mechanism. RpoN is another alternative sigma factor, which in combination with enhancer-binding protein, Rrp2, activates \textit{rpoS} transcription (Figure 2). 
		
		
				\begin{figure}[h]
					% the options are h = here, t = top, b = bottom, p = page of figures. 
					% you can add an exclamation mark to make it try harder, and multiple
					% options if you have an order of preference, e.g.
					% \begin{figure}[h!tbp]
					
					\centering
					% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
					\includegraphics[width = \textwidth]{Figures_Images/rposregulation}
					\caption[RpoS Regulation]{Scheme of RpoS regulation in \textit{Borrelia burgdorferi}.}
					% square brackets here correspond to what is written in the list of figures
					\label{RpoSReg}
					% this is label for the figure
				\end{figure}
				
		
		Rrp2 is activated by phosphorylation from acetyl phosphate, which is an intermediate in the acetate kinase-phosphate acetyltransferase pathway (Samuels, 2011; Xu et al., 2010). Increased temperature and decreased pH increase the amount of the \textit{Borrelia burgdorferi} homolog of carbon storage regulator A, an RNA binding protein. CsrA, through repression of Pta (phosphate acetyltransferase), mediates \textit{rpoS} transcription through the Rrp2-RpoN dependent pathway (Figure 2) (Samuels, 2011; Xu et al., 2010). The long rpoS transcript lacks an RpoN promoter and \textit{rpoN} mutant produces long RpoS transcript. Both \textit{rpoN} and \textit{rpoS} mutants cannot establish mammalian infection. \textit{rpoN} mutants can move into the tick salivary gland from the midgut while \textit{rpoS} mutants cannot. In \textit{Borrelia burgdorferi}, DsrA is a small RNA that can sense an increase in temperature and at low cell density posttranscriptionally regulates the long \textit{rpoS} mRNA transript but not the short. Hfq is RNA chaperone for DsrA, but is also required for mammalian infection, suggesting it has other functions (Samuels, 2011). 
		
		\section{BosR Function and Structure}
		
	  BosR was originally identified in \textit{Borrelia burgdorferi} as a Fur family member based on its sequence similarity to Fur, a regulator of iron uptake. However, it was discovered that \textit{Borrelia burgdorferi} is quite different from other bacteria in that it does not use iron. Based on its homology to PerR, BosR was characterized as a potential \textit{Borrelia burgdorferi} oxidative stress response regulator. There was some debate as to whether that was its main function, since mutants of BosR did not show significantly greater sensitivity to reactive oxygen species compared to wild-type (Samuels, 2011). Some researchers found that BosR serves as a transcriptional activator of the oxidative stress response, while others found it to be a repressor (Boylan et al. 2003;  Katona et al., 2004). Contradictory evidence seems to be a theme in the literature surrounding BosR, resulting in ambiguous conclusions. However, BosR is known to play an important role in \textit{Borrelia burgdorferi} gene regulation. 
		% ask CUS how to change spacing between periods used for et al. to be normal spaces not end of sentence spaces
		
		
		BosR has been shown to be involved either directly or indirectly in the regulation of many genes in the \textit{Borrelia burgdorferi} genome. Without functional BosR, mammalian infection cannot be established. Research agrees that BosR mediates this phenotype through direct regulation of alternative sigma factor RpoS expression. RpoS activates transcription of \textit{ospC} and represses transcription of \textit{ospAB}. As noted previous, RpoS  participates in the expression of over a hundred genes (Samuels, 2011). In light of this, through direct regulation of RpoS, BosR indirectly regulates many genes in the \textit{Borrelia burgdorferi} genome. Microarray analysis of a \textit{bosR} mutant compared to wild-type B31 revealed that BosR differentially regulates 199 genes, upregulates 137 genes, and downregulates 62 genes (Ouyang et al., 2009).  
		
		
		
		
		After determining the critical role BosR plays in regulation of genes that encode virulence factors, researchers investigated the mechanism behind regulation.  BosR binds in the promoter region of \textit{napA}, a gene that encodes an oxidative stress related protein. However, DTT and Zn\textsuperscript{ 2+} are required  for optimal binding to \textit{napA} (Boylan, Posey \& Gherardini, 2003). Consistent with these findings, BosR has been shown to have 1.4 mol of zinc per mol of protein and that even after chelation some zinc has persisted.
		
		
		
	The Norgard lab has investigated the mechanism by which BosR serves as a positive transcriptional activator of \textit{rpoS}, an alternative sigma factor ()Ouyang et al., 2011). Based on \textit{in silico} analysis that suggested structural similarity in DNA binding domains to PerR protein from \textit{Bacillus subtilis} and Fur protein from \textit{Vibrio cholerae}, they hypothesized that BosR functioned by directly binding \textit{rpoS}. Electrophoretic mobility shift assays (EMSAs) were performed with labeled DNA that encompassed a 277 bp region upstream of \textit{rpoS} and a 245 bp region in the \textit{rpoS} encoding region. Recombinant BosR was found to bind the sequence in a dose-dependent manner and showed multiple bands, which suggests that more than one binding site exists within the sequence. DNase I footprinting supports the existence of three binding sites. BosR bound to the direct repeat sequence \-(TAAATTAAAT) upon sequence analysis of the three regions identified by footprinting. This sequence was present in one perfect and one imperfect direct repeat in two of the regions and one imperfect direct repeat in the third region. Mutations to the direct repeat sequence affect binding, in some cases completely abolishing it and in others simply decreasing it.  Approximate dissociation constants for BosR binding sites BS1 and BS2, are 210.2 nM and 36.6 nM, respectively. Binding constants were determined  based on gel image results of showing the amount of DNA bound when different amounts of BosR protein are added (Ouyang et al., 2011).
		
		
		
		 DNase I digestion, did not show that BosR protects regions upstream of \textit{rpoS}, but found that it protects an upstream region of \textit{bosR}, suggesting it may auto-regulate its own transcription (Wang et al., 2013). EMSAs show that BosR recognizes the Fur and Per box, which unsurprisingly, both contain many As and Ts (Wang et al., 2013). 
	
		In 2014, the Norgard revised their hypothesized BosR core binding sequence. Instead the sequence \-(ATTTAANTTAAAT) is predicted to be a putative 6-1-6 inverted repeat that serves as the BosR box. This conclusion, however, was also made solely through mutational analysis and subsequent EMSA results. 
		
	$*$Will add more here about DNA binding$*$
	 
	 
	 The predicted structure of BosR is based on its sequence similarity to Fur family members, Fur, PerR and Zur. Conserved domains include the helix turn helix DNA binding domain, which consists of a three-winged helix with a pocket for DNA, and four cysteine residues that make up the structural zinc binding site. The structural zinc binding site is highly conserved, while two other possible metal binding sites are variable (Gilston et al., 2014) .  
	 
	 $*$Will write more about structure and talk to you. Also fill out captions$*$
	 \clearpage
		
	
		 	\begin{figure}[t]
		 		% the options are h = here, t = top, b = bottom, p = page of figures.
		 		% you can add an exclamation mark to make it try harder, and multiple
		 		% options if you have an order of preference, e.g.
		 		% \begin{figure}[h!tbp]
		 		
		 		\centering
		 		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
		 		\includegraphics[width = \textwidth]{Figures_Images/finalBosRmonomer}
		 		\caption[Predicted structure BosR Monomer]{Structure of BosR monomer based on Phyre2 prediction given the template 4MTD}
		 		% square brackets here correspond to what is written in the list of figures
		 		\label{BosRMonomer}
		 		% this is label for the figure
		 		\end{figure}
		 		
		 		 	\begin{figure}[h]
		 		 		% the options are h = here, t = top, b = bottom, p = page of figures. 
		 		 		% you can add an exclamation mark to make it try harder, and multiple
		 		 		% options if you have an order of preference, e.g.
		 		 		% \begin{figure}[h!tbp]
		 		 		
		 		 		\centering
		 		 		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
		 		 		\includegraphics[width = \textwidth]{Figures_Images/strucznbindingBosR}
		 		 		\caption[BosR Structural $Zn^{2+}$ Binding Site]{BosR structural $Zn^{2+}$ binding site as predicted by Phyre2 and based on alignment of \textit{E. coli} Zur protein and BosR.}
		 		 		% square brackets here correspond to what is written in the list of figures
		 		 		\label{StrucZnBosR}
		 		 		% this is label for the figure
		 		 	\end{figure}
		 		 	
		 		
		 \clearpage
		 			\begin{figure}[h!tbp]
		 				% the options are h = here, t = top, b = bottom, p = page of figures. 
		 				% you can add an exclamation mark to make it try harder, and multiple
		 				% options if you have an order of preference, e.g.
		 				% \begin{figure}[h!tbp]
		 				
		 				\centering
		 				% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
		 				\includegraphics[width = \textwidth]{Figures_Images/ZurDNA}
		 				\caption[Structure of \textit{E. coli} Zur Protein Bound to DNA]{Structure of \textit{E. coli} Zur protein bound to DNA.}
		 				% square brackets here correspond to what is written in the list of figures
		 				\label{ZurDNA}
		 				% this is label for the figure
		 			\end{figure}
		 	\clearpage
		
		 
		 		 	\begin{figure}[h]
		 		 		% the options are h = here, t = top, b = bottom, p = page of figures. 
		 		 		% you can add an exclamation mark to make it try harder, and multiple
		 		 		% options if you have an order of preference, e.g.
		 		 		% \begin{figure}[h!tbp]
		 		 		
		 		 		\centering
		 		 		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
		 		 		\includegraphics[width = \textwidth]{Figures_Images/proteinalignment}
		 		 		\caption[BosR Protein Multiple Alignment]{Multiple alignment of protein sequences that are most similiar to BosR. Structural characterizations correspond to the results of Phyre2 prediction of BosR with the template $4MTD$ (\textit{E. coli} Zur).}
		 		 		% square brackets here correspond to what is written in the list of figures
		 		 		\label{Alignnment}
		 		 		% this is label for the figure
		 		 	\end{figure}
		 	 
	
	
	%texshade, msa alignment r, pymol, phyre (The Phyre2 web portal for protein modeling, prediction and analysis Kelley LA et al. Nature Protocols 10, 845-858 (2015))
	
	
%\LaTeX\ does a great job of formatting tables and paragraphs. Its line-breaking algorithm was the subject of a PhD.\thinspace thesis. It does a fine job of automatically inserting ligatures, and to top it all off it is the only way to typeset good-looking mathematics.
		 
		  
\section{Aims of this Work}

Determining the DNA binding sequence of BosR is critical for identification of other genes that BosR might regulate. 	 Given the amount of conflicting evidence from multiple research groups, a consensus target sequence for BosR has not been clearly identified. Prior work has primarily used mutational or competitive binding approaches to tackle this issue. This thesis employs the CASTing (cyclic amplification and selection of targets) technique to determine the sequence(s) for which BosR has the highest affinity. 


CASTing is an \textit{in vitro} selection tool that utilizes a random DNA library as a starting pool for selection. An immobilized protein of interest is incubated with the DNA library, and the sequences that bind are reserved while others are washed away. PCR is used to amplify the DNA to create the DNA pool for the next round of selection. After four to six cycles of this process, the remaining DNA pool should contain only sequences that bind with high affinity, hopefully leading to a consensus sequence. EMSAs can be performed for each cycle and the DNA  can be cloned and sequenced. 
		 	 	\begin{figure}[h]
		 	 		% the options are h = here, t = top, b = bottom, p = page of figures.
		 	 		% you can add an exclamation mark to make it try harder, and multiple
		 	 		% options if you have an order of preference, e.g.
		 	 		% \begin{figure}[h!tbp]
		 	 		
		 	 		\centering
		 	 		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
		 	 		\includegraphics{Figures_Images/DNA_library_creation}
		 	 		\caption[DNA Library Creation]{Scheme of PCR to create DNA Library}
		 	 		 % square brackets here correspond to what is written in the list of figures
		 	 		\label{DNALibrary}
		 	 		 % this is label for the figure
		 	 	\end{figure}
		 	 	
		 	\begin{figure}[h]
		 		% the options are h = here, t = top, b = bottom, p = page of figures.
		 		% you can add an exclamation mark to make it try harder, and multiple
		 		% options if you have an order of preference, e.g.
		 		% \begin{figure}[h!tbp]
		 		
		 		\centering
		 		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
		 		\includegraphics[width = \textwidth]{Figures_Images/Blown_up_Casting}
		 		\caption[Binding Reaction and Immunoprecipitation Scheme ]{CASTing scheme}
		 		\label{CASTingScheme}
		 	\end{figure}
		 	
		 	$*$ will fill out captions and add surface entropy reduction mutation goals.$*$
		 
%\section*{Personal Thesis Notes From Reading with Refs}
%"Rather, it contained 1.4 mol of zinc per mol of protein. Moreover, in order to remove bound metal(s) from BosR, we also dialyzed the protein in the presence of 10 mM EDTA. However, 0.3 mol of zinc/mol of proteins remained in the demetallated BosR (Fig. 3D), suggesting that the recombinant protein bound zinc avidly" ICP-AES data (Ouyang, 2011)

%Anyone who needs to use math, tables, a lot of figures, complex cross-references, IPA or who just cares about the final appearance of their document should use \LaTeX. At Reed, math majors are required to use it, most physics majors will want to use it, and many other science majors may want it also.
    \sofiachapter{Materials and Methods}
    \addcontentsline{toc}{chapter}{Materials and Methods}
    \chaptermark{Materials and Methods}
    \markboth{Materials and Methods}{Materials and Methods}
    % The three lines above are to make sure that the headers are right, that the intro gets included in the table of contents, and that it doesn't get numbered 1 so that chapter one is 1.
   \section{Recombinant DNA Constructs}
   
   \subsection{pSMT3 Vector}
   
   	\begin{figure}[h]
   		% the options are h = here, t = top, b = bottom, p = page of figures. 
   		% you can add an exclamation mark to make it try harder, and multiple
   		% options if you have an order of preference, e.g.
   		% \begin{figure}[h!tbp]
   		
   		\centering
   		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
   		\includegraphics[width = 0.5\textwidth]{Figures_Images/pET_SUMO}
   		\caption[pET SUMO Vector with BosR Insert]{The pET SUMO vector with BosR gene insert. Figure borrowed from Levitz, 2014. fig cap from tallys thesis = Map of the pSMT3 vector containing a BosR sequence used in this study.
   			Vector map partially created using SnapGene Viewer™.}
   		% square brackets here correspond to what is written in the list of figures
   		\label{pETSUMOBosR}
   		% this is label for the figure
   	\end{figure}
   	
   The pET SUMO vector, pSMT3, was used for all of the constructs in this study (Figure ~\ref{pETSUMOBosR}). An N terminal His$_{6}$ tagged SUMO (small ubiquitin like modifier protein) is located upstream of the gene of interest (GOI). The His-SUMO-GOI is downstream of a T7 promoter and  under control of a \textit{lac} operator. Addition of isopropyl $\beta$-D-1-thiogalactopyranoside (IPTG) inhibits activity of the \textit{lac} repressor, allowing for the expression of the construct. The hexahistidine tag aides in purifying the protein using immobilized metal ion affinity chromatography (IMAC). The His-SUMO domain can be cleaved from the protein of interest by ULP-1 (Ubl-specific protease 1). The pSMT3 vector also contains a kanamycin resistance gene for plasmid selection. All plasmids in this study were sequenced through genscript to confirm desired inserts. 
   
   \subsection{\textit{E. coli} Strains}
   
   DH5$\alpha$ and NiCo21 competent \textit{E. coli} cells (New England Biolabs) were used in this study. Plasmid was transformed into DH5$\alpha$ cells, which do not contain a T7 RNA polymerase, when preparing samples for sequencing or making more plasmid. Plasmid was transformed into NiCo21 cells containing a T7 RNA polymerase when creating glycerol stocks for plasmid expression. 
   
   \subsection{\textit{E. coli} Growth Conditions}
   \textit{E. coli} cells were grown in sterile Luria  Broth (LB) (25 g per L DI water). All cells in this study were incubated at 37 $^\circ$C. Shaking incubations were at ~200-250 RPM
   
   \subsection{Transformation}
   Plasmid was transformed into competent cells. Aliquots of competent cells (50 $\mu$l) stored at -80 $^\circ$C were thawed on ice for 15 minutes. Plasmid (1 or 5 $\mu$l, depending on plasmid prep) was added to cell pellet at the bottom of the tube. Cells were let rest on ice for 30 minutes. Cells were incubated in a 42 $^\circ$C water bath for 30 seconds. Sterile LB (950 $\mu$l) was added to the cells. Cells were put in a shaking incubator (200 RPM) at 37 $^\circ$C for 1 hour and then 250 $\mu$l of culture was plated on LB agar plates containing kanamycin. Plates were allowed to dry and put in a stationary incubator at 37 $^\circ$C overnight. 
   
   \subsection{Plasmid Mini-Prep}
   Overnight cultures were spun down for 10 minutes at 4 $^\circ$C between 2000-3260 xg. Supernatant was carefully decanted off and 1.2 mL of sterile H$_{2}$O was added. Pellet was resuspended in water and 600 $\mu$l was used with Zymogen mini-prep kit. Kit instructions were followed to isolate plasmid, with the exception of an additional wash with zyppy wash buffer. 
   
   
   
   
   \section{Polymerase Chain Reaction}
   
   \subsection{Amplification of Ran76 for CASTing}
   
\begin{table}[H]
	\caption{Conditions for Ran76 PCR to Create Pool of Random DNA Sequences and Conditions of qPCR to Quantify Amount of Recovered Ran76} 
	\label{Ran76PCRandqPCR}
	\begin{tabular}{|l | l| |l| l|}
		\hline
		Ran76 PCR & Volume ($\mu$l) & Ran76 qPCR & Volume ($\mu$l) \\
		\hline 
		PrimF (10 $\mu$M) & 1 & PrimF (10 $\mu$M) & 1   \\ 
		Primer R (10 $\mu$M) & 1  & Primer R (10 $\mu$M) & 1  \\  			
		ds Ran76 (1 pg/$\mu$l) & 10  & Recovered ds Ran76 &  1  \\
		Taq Buffer (10X) & 5 & Sterile H$_{2}$O & 6.4 \\ 
		MgCl$_{2}$ (25 mM) & 3 & DMSO & 0.6 \\  
		dNTPs (10 mM) & 1 & 2X Phusion & 10 \\     
		Sterile H$_{2}$O & 28.75  & & \\
		Taq Polymerase (5 units/$\mu$l) & 0.25 & & \\
		\hline   
		Total Volume & 50 & Total Volume & 20  \\
		\hline
	\end{tabular}
\end{table}

Cycling protocol for Ran76 is initial denaturation for 30 seconds at 95 $^{\circ}$C, denaturation for 15 seconds at 95 $^{\circ}$C, annealing for 15 seconds at 60 $^{\circ}$C, elongation for 10 seconds at 68 $^{\circ}$C, repeat from denaturation 29 times, final extension for 5 minutes at 68 $^{\circ}$C. 

 \subsection{Creation of MntR-FLAG}

\begin{table}[H]
	\caption{PCR Conditions to Create MntR-FLAG} 
	\label{MntRFLAGPCR}
	\begin{tabular}{|l | l| |l| l|}
		\hline
		MntR-FLAG PCR 1 & Volume ($\mu$l) & MntR-FLAG PCR 2  & Volume ($\mu$l) \\
		\hline 
		MntRFF (50 $\mu$M) & 0.5 & MntRFF (50 $\mu$M) & 0.5   \\ 
		MntRFR1 (50 $\mu$M) & 0.5 & MntRFR3 (50 $\mu$M) & 0.5 \\  			
		pSMT3 MntR & 1  & MntR-FLAG PCR 1 &  1  \\
		Taq Buffer (10X) & 5 & Sterile H$_{2}$O & 6.4 \\ 
		MgCl$_{2}$ (25 mM) & 3 & DMSO & 0.6 \\  
		dNTPs (10 mM) & 1 & 2X Phusion & 10 \\     
		Sterile H$_{2}$O & 7.4  & Sterile H$_{2}$O & 0  \\
		DMSO & 0.6 &  &  \\
		2X Phusion & 10 & 2X Phusion & 10 \\
		\hline   
		Total Volume & 20 & Total Volume & 20  \\
		\hline
	\end{tabular}
\end{table}

Cycling protocol for MntR-FLAG PCR is initial denaturation for 30 seconds at 98 $^{\circ}$C, denaturation for 10 seconds at 98 $^{\circ}$C, annealing for 10 seconds at 60 $^{\circ}$C, elongation for 30 seconds at 72 $^{\circ}$C, repeat from denaturation 29 times, final extension for 5 minutes at 72 $^{\circ}$C. 


 \subsection{Site-directed Mutagenesis of BosR}
\begin{table}[H]
	\caption{BosR Mutagenesis PCR} 
	\label{BosRmutpcr}
	\begin{tabular}{|l| | l|}
		\hline
		BosR Mutagenesis PCR & Volume ($\mu$l)  \\
		\hline 
		AAAf (50 $\mu$M) & 0.5   \\ 
		AAAr (50 $\mu$M) & 0.5 \\  			
		pSMT3 WT BosR or CNDT BosR & 1   \\
		Sterile H$_{2}$O & 7.0   \\ 
		DMSO & 1.0 \\  
		2X Phusion & 10  \\     
		\hline   
		Total Volume & 20  \\
		\hline
	\end{tabular}
\end{table}

Cycling protocol for BosR mutagenesis PCR is initial denaturation for 30 seconds at 98 $^{\circ}$C, denaturation for 10 seconds at 98 $^{\circ}$C, annealing for 10 seconds at 50 $^{\circ}$C, elongation for 3 minutes at 72 $^{\circ}$C, repeat from denaturation 29 times, final extension for 5 minutes at 72 $^{\circ}$C. 


\section{Gel Purification of DNA}

MntR-FLAG PCR 1 was gel purified using the Zymo Clean Gel DNA Recovery Kit according to kit instructions. DNA was eluted with 15 \micro l of sterile $H_{2}O$.

\section{Gibson Cloning}
MntR-FLAG PCR product was cloned into a linearized pSMT3 plasmid using the Gibson Assembly master mix. Three times excess of insert (MntR-FLAG PCR 2 product) to plasmid was used. Gibson Assembly reaction was performed in a thermocycler set to 50 $^\circ$C for 20 minutes. 


 
\section{Agarose Gel Electrophoresis}

Gels to visualize Ran76 DNA contained 3\% agarose with 0.5x TBE (Tris-borate-EDTA) buffer. Gels to visualize all other DNA in this study contained 1.5\% agarose with 1x TAE (Tris-acetate-EDTA) buffer. All gels in this study contained 1x SYBR Safe DNA stain from Invitrogen. Gels were run at $\sim$ 100 V. 

\section{Protein Purification}

\subsection{MntR-FLAG}

An overnight culture (10 mL LB + 10 \micro l 50 \micro g/mL) of cells expressing pSMT3-MntR-FLAG were inoculated from a glycerol stock and grown at 37 $^\circ$C, shaking at $\sim$ 250 RPM. Liter cultures containing 50 \micro g/L kanamycin were inoculated with 5 mL of overnight culture and grown at 37 $^\circ$C, shaking at $\sim$ 250 RPM until OD between 0.06-0.1 was reached. Cells were induced with 0.15 g IPTG and returned to the shaking incubator for 3 hours. Cultures were spun down at 5000 RPM in big centrifuge for 10 minutes. Pellets were scraped out and the centrifuge bottles were rinsed with $\sim$ 20 mL of chilled MntR lysis buffer (25 mM HEPES, 300 mM NaCl, 10 mM imidazole, 5\% v/v glycerol, pH 7.5). Tube was spun down at 4 $^\circ$C for 10 minutes at 3850 xg and buffer was poured off. Pellet was either stored at -80 $^\circ$C or resuspended in 25 mL chilled MntR lysis buffer. PMSF (phenyl-methyl-sulfonyl chloride, 2.5 mg) and 15.5 mg lysozyme were added to the buffer. Cells were lysed by sonication (see Appendix for protocol). Lysed cells were centrifuged for 20 min, 12,000 RPM in SS-34 rotor at 4 $^\circ$C. Lysate was loaded onto a Co$^{2+}$ column pre-equilibrated with 50 mLs of MntR lysis buffer. The flow through was pumped out at 1.5 mL/min and 5 minute fractions were collected. The column was washed with 50 mL of MntR lysis buffer and eluted with 50 mLs of MntR elution buffer (25 mM HEPES, 300 mM NaCl, 300 mM imidazole, 5\% v/v glycerol, pH 7.5). A dot blot of fractions was used to determine which fractions to combine for dialysis. Fractions 10-13 were pooled and dialyzed with 50 \micro l of Ulp against 1 L of MntR dialysis buffer (25 mM HEPES, 300 mM NaCl, 10\% v/v glycerol, pH 7.5) overnight. The protein was added to a Ni$^{2+}$ column pre-equilibrated with chilled MntR lysis buffer. The flow rate was 1.5 mL/min and 5 minute fractions were collected. The column was washed with 30 mLs of MntR lysis buffer and eluted with 50 mLs MntR elution buffer. Fractions 2-7 were combined after analysis by dot blot and dialyzed overnight against MntR dialysis buffer with 75 \micro l $\beta$-mercaptoethanol. The buffer was changed twice more and the protein was concentrated using an Amicon filter... . 


\section{Fluorescence Anisotropy}
All FA experiments were performed with the Beacon... The machine was blanked on 1 mL of buffer. Three recordings were taken after adding 1 \micro l of 1 \micro M fluorinated DNA to the tube. Intensity was monitored to ensure a decrease and the last mP value was used in the analysis as a zero point. Protein was added in the following increments: 0.5, 0.6, 0.9, 1.2, 1.8, 3.4, 4.7, 7.3, 11.6, 18 \micro l. Readings were taken after 30 seconds and samples were gently vortexed after each addition. * Add what filter was used *  

\section{CASTing}

\begin{figure}[[h!tbp]
	% the options are h = here, t = top, b = bottom, p = page of figures. 
	% you can add an exclamation mark to make it try harder, and multiple
	% options if you have an order of preference, e.g.
	% \begin{figure}[h!tbp]
	\centering
	% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
	\includegraphics[width = \textwidth]{Figures_Images/Casting_cycle.pdf}
	\caption[CASTing Overal Cycle Scheme]{CASTing Process}
	% square brackets here correspond to what is written in the list of figures
	\label{overallcastingscheme}
	% this is label for the figure
\end{figure}

\begin{figure}[[h!tbp]
	% the options are h = here, t = top, b = bottom, p = page of figures. 
	% you can add an exclamation mark to make it try harder, and multiple
	% options if you have an order of preference, e.g.
	% \begin{figure}[h!tbp]
	\centering
	% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
	\includegraphics[width = \textwidth]{Figures_Images/Casting_Limited}
	\caption[CASTing Selection Scheme]{Selection Step of CASTing Process}
	% square brackets here correspond to what is written in the list of figures
	\label{Selectionscheme}
	% this is label for the figure
\end{figure}

\clearpage


\section{Gel Shift and Gel Supershift Assay}

Binding reaction conditions include the following: fluorinated DNA (F-DNA) with expected specific binding alone, with antibody, with protein, with protein and antibody, non-binding F-DNA alone, with antibody, with protein and antibody. All binding reactions were performed in binding buffer (pH 7.4, 10 mM Tris, 5 mM NaCl, 50 mM KCl, 50 \micro g/mL BSA, 1 mM DTT; 1 mM MnCl for use with MntR-FLAG) and contained $\sim$ 100 ng of fluorinated DNA and 1 \micro g of poly dI$\cdot$dC. For the relevant conditions 1 \micro g of protein and/or 1 \micro g of FLAG antibody was added.

Binding reactions were performed at room temperature for 30 minutes. Antibody was added and then incubated for another 30 minutes at room temperature. Glycerol (2.5\% v/v) was added to binding reactions prior to gel loading. 

DNA and DNA-protein complexes were run on 5\% non-denaturing polyacrylamide gels in either Tris-borate buffer (MntR-FLAG) or Tris-borate-EDTA buffer (BosR-FLAG) at 4 $^\circ$C and $\sim$ 35 V for $\sim$ 5 hours. 
  
 

   
\sofiachapter{Results}
  
  
  \section{BosR-FLAG}
  
         	\begin{figure}[h]
         		% the options are h = here, t = top, b = bottom, p = page of figures. 
         		% you can add an exclamation mark to make it try harder, and multiple
         		% options if you have an order of preference, e.g.
         		% \begin{figure}[h!tbp]
         		\centering
         		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
         		\includegraphics[width = \textwidth]{Figures_Images/FA_BosR_poly.pdf}
         		\caption[Determining DNA Activity of BosR-FLAG by Fluorescence Anistropy]{Fluorescence Anistropy for BosR-FLAG with F-SREzy and 2 $\mu$g of poly dI$\cdot$dC in CASTing Binding Buffer  a $K_{d}$ of 0.97 $\pm$ 0.09 $\mu$M based on fit to the equation \FAstdfit , where P refers to protein concentration.}
         		% square brackets here correspond to what is written in the list of figures
         		\label{BosRFLAGFApoly}
         		% this is label for the figure
         	\end{figure}
  
  
  \section{CASTing} 
  
    	%\begin{figure}[h]
    		% the options are h = here, t = top, b = bottom, p = page of figures. 
    		% you can add an exclamation mark to make it try harder, and multiple
    		% options if you have an order of preference, e.g.
    		 \begin{figure}[h!tbp]
    		\centering
    		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
    		\includegraphics[width = 0.5\textwidth]{Figures_Images/PCR_9_26_15.pdf}
    		\caption[PCR to Create Pool of Random DNA]{PCR Ran 76 with 10 pg of Ran76 template cleaned and concentrated}
    		% square brackets here correspond to what is written in the list of figures
    		\label{PCRRan76}
    		% this is label for the figure
    	\end{figure}
 
 
 \begin{figure}[h!tbp]
 	\centering
 	% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
 	\includegraphics[width =\textwidth]{Figures_Images/SC_qPCR_cycle1_Ran76_Fluor_curves.pdf}
 	\caption[qPCR of 1st Cycle of CASTing Fluorescence Curves]{qPCR performed with different templates}
 	% square brackets here correspond to what is written in the list of figures
 	\label{qPCRcycle1fluorcurves}
 	% this is label for the figure
 \end{figure}

 
 
      	%\begin{figure}[h]
      		% the options are h = here, t = top, b = bottom, p = page of figures. 
      		% you can add an exclamation mark to make it try harder, and multiple
      		% options if you have an order of preference, e.g.
      		 \begin{figure}[h!tbp]
      		\centering
      		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
      		\includegraphics[width = 0.5\textwidth]{Figures_Images/qPCR_cycle1_10_19_15.pdf}
      		\caption[qPCR of 1st Cycle of CASTing Gel Analysis]{qPCR samples run on 3\% agarose/TBE gel. Lane L contains 2 $\mu$l of 2-log ladder (NEB), lanes 1-6 are qPCR reactions with various templates. Template lane 1, $H_{2}O$, lane 2, poly dI $\cdot$ dC, lane 3, DNA from binding reaction without BosR-FLAG, lane 4, DNA from binding reaction without FLAG antibody, lane 5 DNA from binding reaction with 5 $\mu$g of both BosR-FLAG and FLAG antibody, lane 6, 2 pg standard of ds Ran76.}
      		% square brackets here correspond to what is written in the list of figures
      		\label{qPCRcycle1}
      		% this is label for the figure
      	\end{figure}
 
 \clearpage
      	
      	
      	
  
  \section{MntR-FLAG}
        	%\begin{figure}[h]
        	% the options are h = here, t = top, b = bottom, p = page of figures. 
        	% you can add an exclamation mark to make it try harder, and multiple
        	% options if you have an order of preference, e.g.
        	\begin{figure}[h!tbp]
        		\centering
        		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
        		\includegraphics[width = \textwidth]{Figures_Images/SC_PCR_MntRFLAG.pdf}
        		\caption[PCR to Create MntR-FLAG]{PCR to check MntR in pSMT3 plasmid. PCR 1 and 2 for final construct.}
        		% square brackets here correspond to what is written in the list of figures
        		\label{PCRMntRFLAG}
        		% this is label for the figure
        	\end{figure}
     
        	\begin{figure}[h!tbp]
        		\centering
        		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
        		\includegraphics[width = \textwidth]{Figures_Images/MntR_FLAG_Protein.pdf}
        		\caption[MntR-FLAG Purification]{SDS Page Gel of fractions for MntR-FLAG purification}
        		% square brackets here correspond to what is written in the list of figures
        		\label{PurificationMntRFLAG}
        		% this is label for the figure
        	\end{figure}
     
       	\begin{figure}[h!tbp]
       		% the options are h = here, t = top, b = bottom, p = page of figures. 
       		% you can add an exclamation mark to make it try harder, and multiple
       		% options if you have an order of preference, e.g.
       		% \begin{figure}[h!tbp]
       		\centering
       		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
       		\includegraphics[width = \textwidth]{Figures_Images/MntR_FLAG_FA.pdf}
       		\caption[Determining DNA Activity of MntR-FLAG by Fluorescence Anistropy]{Fluorescence Anistropy for MntR-FLAG with 1 mM Mn$^2+$ F-M21 with a $K_{d}$ of 29 $\pm$ 3 nM based on fit to the equation \FAstdfit , where P refers to protein concentration.}
       		% square brackets here correspond to what is written in the list of figures
       		\label{MntRFLAGFA}
       		% this is label for the figure
       	\end{figure}
      
      %only table example
 \begin{table}[ht]
 	\caption{Coefficients for Standard Binding Fit for MntR-FLAG bound to F-M21 DNA Sequence} 
 	\label{MntRFLAGCoef}
 	\begin{tabularx}{3.0in}{|X|X|X|}
 		\hline
 		\textbf{ } & \textbf{Estimate} & \textbf{SE} \\ 
 		\hline
 		$\Delta$r & 0.027 & 0.0009 \\ 
 		\hline
 		$K_{d}$ ($nM$) &  29  &  3  \\ 
 		\hline
 		$r_{min}$ & 0.028 & 0.0008 \\ 
 		\hline
 	\end{tabularx}
 \end{table}
 
 \begin{table}[ht]
 \centering
 \parbox{5.0in}{\caption{Error associated with Specific Binding Fit}} 
 \begin{tabularx}{3.0in}{|X|X|X|}
 	\hline
 	\textbf{Residual Std. Error} & \textbf{Deviance} & \textbf{Variance of Residuals} \\ 
 	\hline
 	8.719e-04 & 6.082e-06 & 6.082e-07 \\ 
 	\hline
 \end{tabularx}
\end{table}


Add gelshift image 

\clearpage

   \section{BosR Reduced Surface Entropy}
   
   insert site directed mutagenesis pcr for wt and cndt
   
    		 \begin{figure}[h!tbp]
    		 	\centering
    		 	% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
    		 	\includegraphics[width = 0.5\textwidth]{Figures_Images/SC_BosR_Mut_PCR.pdf}
    		 	\caption[BosR QKE --> AAA Site Directied Mutagenesis PCR Gel]{PCR of pSMT3-BosR WT and CNDT for site directed mutagenesis, QKE to AAA. gradient}
    		 	% square brackets here correspond to what is written in the list of figures
    		 	\label{BosRmutPCR}
    		 	% this is label for the figure
    		 \end{figure}
  
   
\appendix
\sofiachapter{Summer Research} 
   
           	\begin{figure}[h]
           		% the options are h = here, t = top, b = bottom, p = page of figures. 
           		% you can add an exclamation mark to make it try harder, and multiple
           		% options if you have an order of preference, e.g.
           		% \begin{figure}[h!tbp]
           		\centering
           		% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
           		\includegraphics[width = \textwidth]{Figures_Images/FA_BosR_active_sum.pdf}
           		\caption[Determining DNA Activity of BosR-FLAG by Fluorescence Anistropy]{Fluorescence Anistropy for BosR-FLAG with F-SREzy in TL BosR FA Buffer  a $K_{d}$ of 0.99 $\pm$ 0.09 $\mu$M based on fit to the equation \FAstdfit , where P refers to protein concentration.}
           		% square brackets here correspond to what is written in the list of figures
           		\label{BosRFLAGFAactive}
           		% this is label for the figure
           	\end{figure}
           	
           	
       		 \begin{figure}[h!tbp]
       		 	\centering
       		 	% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
       		 	\includegraphics[width =\textwidth]{Figures_Images/SC_Std_Fluorescence}
       		 	\caption[qPCR Fluorescence Curves for Ran76 Standards]{qPCR Fluorescence Curves for Ran76 Standards. From left to right increasing concentration 0.002 ng, 0.0002 ng etc...}
       		 	% square brackets here correspond to what is written in the list of figures
       		 	\label{qPCRRan76Stds}
       		 	% this is label for the figure
       		 \end{figure}
       		 
       		 
       		     		 \begin{figure}[h!tbp]
       		     		 	\centering
       		     		 	% DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
       		     		 	\includegraphics[width =\textwidth]{Figures_Images/SC_Std_curve}
       		     		 	\caption[qPCR Standard Curves for Ran76 Amplification]{qPCR standard curve for Ran76. From left to right decreasing concentration  to 0.2 ng}
       		     		 	% square brackets here correspond to what is written in the list of figures
       		     		 	\label{qPCRRan76Stdcurve}
       		     		 	% this is label for the figure
       		     		 \end{figure}
           	
           		
    \sofiachapter{Supplemental Methods}
    
    	\begin{table}[!hp]
    		\caption{DNA Sequences for Manipulation of DNA} 
    		\label{DNAsequences}
    		\begin{tabular}{|l | p{12cm}|}
    			\hline
    			Label & Sequence \\
    			\hline 
    			PrimF & 5'-GCT GCA GTT GCA CTG AAT TCG CCT C-3'  \\ 
    			\hline
    			Primer R  & 5'-CAG GTC AGT TCA GCG GAT CCT GTC G-3' \\  			
    			\hline
    			MntRFF & 5'-ACA GAG AAC AGA TTG GTG GAA CAA CAC CAA G-3'    \\
    			\hline
    			MntRF1  & 5'-GTC GTC GTC GTC TTT GTA GTC CTG ATT ATG TTC TGT TTT CTT TTG GAT TG-3'  \\ 
    			\hline
    			MntRF3 & 5'CGA AGT GCG GCC GCA AGC TTT TAT TTG TCG TCG TCG TCT TTG TAG TCC TG-3'  \\  
    			\hline
    			AAAf & 5'-ACT GAT GCA GCA GCA ACA AAA TTT TAT CTA AGC TTG-3'  \\     
    			\hline
    			AAAr & 5'-TTT TGT TGC TGC TGC ATC AGT AGT TTT TAT ATC TTT TAG-3' \\
    			\hline
    			F-M21 & 5'-F-GAG TTT CCT TAA GGC AAA TTG-3'  \\
    			\hline
    			MDR1 & *need to find seq* \\  
    			\hline
    			SRE & *need to find seq*  \\
    			\hline
    		\end{tabular}
    	\end{table}

\clearpage
\section{References, Labels, Custom Commands and Footnotes}
It is easy to refer to anything within your document using the \texttt{label} and \texttt{ref} tags.  Labels must be unique and shouldn't use any odd characters; generally sticking to letters and numbers (no spaces) should be fine. Put the label on whatever you want to refer to, and put the reference where you want the reference. \LaTeX\ will keep track of the chapter, section, and figure or table numbers for you. 

\subsection{References and Labels}
Sometimes you'd like to refer to a table or figure, e.g. you can see in Figure \ref{subd2} that you can rotate figures . Start by labeling your figure or table with the label command (\verb=\label{labelvariable}=) below the caption (see the chapter on graphics and tables for examples). Then when you would like to refer to the table or figure, use the ref command (\verb=\ref{labelvariable}=). Make sure your label variables are unique; you can't have two elements named ``default." Also, since the reference command only puts the figure or table number, you will have to put  ``Table" or ``Figure" as appropriate, as seen in the following examples:

 As I showed in Table \ref{inheritance} many factors can be assumed to follow from inheritance. Also see the Figure \ref{subd} for an illustration.
 
\subsection{Custom Commands}\label{commands}
Are you sick of writing the same complex equation or phrase over and over? 

The custom commands should be placed in the preamble, or at least prior to the first usage of the command. The structure of the \verb=\newcommand= consists of the name of the new command in curly braces, the number of arguments to be made in square brackets and then, inside a new set of curly braces, the command(s) that make up the new command. The whole thing is sandwiched inside a larger set of curly braces. 
% Note: you cannot use numbers in your commands!
\newcommand{\hydro}{H$_2$SO$_4$}


In other words, if you want to make a shorthand for H$_2$SO$_4$, which doesn't include an argument, you would write: \verb=\newcommand{\hydro}{H$_2$SO$_4$}= and then when you needed  to use the command you would type \verb=\hydro=. (sans verb and the equals sign brackets, if you're looking at the .tex version). For example: \hydro

\subsection{Footnotes and Endnotes}
	You might want to footnote something.\footnote{footnote text} Be sure to leave no spaces between the word immediately preceding the footnote command and the command itself. The footnote will be in a smaller font and placed appropriately. Endnotes work in much the same way. More information can be found about both on the CUS site.
	
\section{Bibliographies}
	Of course you will need to cite things, and you will probably accumulate an armful of sources. This is why BibTeX was created. For more information about BibTeX and bibliographies, see our CUS site (\url{web.reed.edu/cis/help/latex/index.html})\footnote{\cite{reedweb:2007}}. There are three pages on this topic: {\it bibtex} (which talks about using BibTeX, at \url{/latex/bibtex.html}), {\it bibtexstyles} (about how to find and use the bibliography style that best suits your needs, at \url{/latex/bibtexstyles.html}) and {\it bibman} (which covers how to make and maintain a bibliography by hand, without BibTeX, at at \url{/latex/bibman.html}). The last page will not be useful unless you have only a few sources. There used to be APA stuff here, but we don't need it since I've fixed this with my apa-good natbib style file.
	
\subsection{Tips for Bibliographies}
\begin{enumerate}
\item Like with thesis formatting, the sooner you start compiling your bibliography for something as large as thesis, the better. Typing in source after source is mind-numbing enough; do you really want to do it for hours on end in late April? Think of it as procrastination.
\item The cite key (a citation's label) needs to be unique from the other entries.
\item When you have more than one author or editor, you need to separate each author's name by the word ``and'' e.g.\\ \verb+Author = {Noble, Sam and Youngberg, Jessica},+.
\item Bibliographies made using BibTeX (whether manually or using a manager) accept LaTeX markup, so you can italicize and add symbols as necessary.
\item To force capitalization in an article title or where all lowercase is generally used, bracket the capital letter in curly braces.
\item You can add a Reed Thesis citation\footnote{\cite{noble:2002}} option. The best way to do this is to use the phdthesis type of citation, and use the optional ``type'' field to enter ``Reed thesis'' or ``Undergraduate thesis''. Here's a test of Chicago, showing the second cite in a row\footnote{\cite{noble:2002}} being different. Also the second time not in a row\footnote{\cite{reedweb:2007}} should be different. Of course in other styles they'll all look the same.
\end{enumerate}
\section{Anything else?}
If you'd like to see examples of other things in this template, please contact CUS (email cus@reed.edu) with your suggestions. We love to see people using \LaTeX\ for their theses, and are happy to help.


\chapter{Mathematics and Science}	
\section{Math}
	\TeX\ is the best way to typeset mathematics. Donald Knuth designed \TeX\ when he got frustrated at how long it was taking the typesetters to finish his book, which contained a lot of mathematics. 
	
	If you are doing a thesis that will involve lots of math, you will want to read the following section which has been commented out. If you're not going to use math, skip over this next big red section. (It's red in the .tex file but does not show up in the .pdf.)
%	
%% MATH and PHYSICS majors: Uncomment the following section	
%	$$\sum_{j=1}^n (\delta\theta_j)^2 \leq {{\beta_i^2}\over{\delta_i^2 + \rho_i^2}}
%\left[ 2\rho_i^2 + {\delta_i^2\beta_i^2\over{\delta_i^2 + \rho_i^2}} \right] \equiv \omega_i^2
%$$

%From Informational Dynamics, we have the following (Dave Braden):

%After {\it n} such encounters the posterior density for $\theta$ is

%$$
%\pi(\theta|X_1< y_1,\dots,X_n<y_n) \varpropto \pi(\theta) \prod_{i=1}^n\int_{-\infty}^{y_i}
%   \exp\left(-{(x-\theta)^2\over{2\sigma^2}}\right)\ dx
%$$

%

%Another equation:

%$$\det\left|\,\begin{matrix}%
%c_0&c_1\hfill&c_2\hfill&\ldots&c_n\hfill\cr
%c_1&c_2\hfill&c_3\hfill&\ldots&c_{n+1}\hfill\cr
%c_2&c_3\hfill&c_4\hfill&\ldots&c_{n+2}\hfill\cr
%\,\vdots\hfill&\,\vdots\hfill&
%  \,\vdots\hfill&&\,\vdots\hfill\cr
%c_n&c_{n+1}\hfill&c_{n+2}\hfill&\ldots&c_{2n}\hfill\cr
%\end{matrix}\right|>0$$

%
%Lapidus and Pindar, Numerical Solution of Partial Differential Equations in Science and
%Engineering.  Page 54

%$$
%\int_t\left\{\sum_{j=1}^3 T_j \left({d\phi_j\over dt}+k\phi_j\right)-kT_e\right\}w_i(t)\ dt=0,
%   \qquad\quad i=1,2,3. 
%$$

%L\&P  Galerkin method weighting functions.  Page 55

%$$
%\sum_{j=1}^3 T_j\int_0^1\left\{{d\phi_j\over dt} + k\phi_j\right\} \phi_i\ dt 
%   = \int_{0}^1k\,T_e\phi_idt, \qquad i=1,2,3 $$
%   
%Another L\&P (p145)

%$$
%\int_{-1}^1\!\int_{-1}^1\!\int_{-1}^1 f\big(\xi,\eta,\zeta\big) 
%   = \sum_{k=1}^n\sum_{j=1}^n\sum_{i=1}^n w_i w_j w_k f\big( \xi,\eta,\zeta\big).
%$$

%Another L\&P (p126)

%$$
%\int_{A_e} (\,\cdot\,) dx dy = \int_{-1}^1\!\int_{-1}^1 (\,\cdot\,) \det[J] d\xi d\eta.
%$$

\section{Chemistry 101: Symbols}
Chemical formulas will look best if they are not italicized. Get around math mode's automatic italicizing by using the argument \verb=$\mathrm{formula here}$=, with your formula inside the curly brackets.

So, $\mathrm{Fe_2^{2+}Cr_2O_4}$ is written \verb=$\mathrm{Fe_2^{2+}Cr_2O_4}$=\\
Exponent or Superscript: O$^{-}$\\
Subscript: CH$_{4}$\\

To stack numbers or letters as in $\mathrm{Fe_2^{2+}}$, the subscript is defined first, and then the superscript is defined.\\
Angstrom: {\AA}\\
Bullet: CuCl $\bullet$ 7H${_2}$O\\
Double Dagger: \ddag \/\\
Delta: $\Delta$\\
Reaction Arrows: $\longrightarrow$ or  $\xrightarrow{solution}$\\
Resonance Arrows: $\leftrightarrow$\\
Reversible Reaction Arrows: $\rightleftharpoons$ or $\xrightleftharpoons[ ]{solution}$ (the latter requires the chemarr package)\\


\subsection{Typesetting reactions}
You may wish to put your reaction in a figure environment, which means that LaTeX will place the reaction where it fits and you can have a figure legend if desired:
\begin{figure}[htbp]
\begin{center}
$\mathrm{C_6H_{12}O_6  + 6O_2} \longrightarrow \mathrm{6CO_2 + 6H_2O}$
\caption{Combustion of glucose}
\label{combustion of glucose}
\end{center}
\end{figure}

\subsection{Other examples of reactions}
$\mathrm{NH_4Cl_{(s)}} \rightleftharpoons \mathrm{NH_{3(g)}+HCl_{(g)}}$\\
$\mathrm{MeCH_2Br + Mg} \xrightarrow[below]{above} \mathrm{MeCH_2\bullet Mg \bullet Br}$

\section{Physics}

Many of the symbols you will need can be found on the math page (\url{http://web.reed.edu/cis/help/latex/math.html}) and the Comprehensive \LaTeX\ Symbol Guide (enclosed in this template download).  You may wish to create custom commands for commonly used symbols, phrases or equations, as described in Chapter \ref{commands}.

\section{Biology}
You will probably find the resources at \url{http://www.lecb.ncifcrf.gov/~toms/latex.html} helpful, particularly the links to bsts for various journals. You may also be interested in TeXShade for nucleotide typesetting (\url{http://homepages.uni-tuebingen.de/beitz/txe.html}).  Be sure to read the proceeding chapter on graphics and tables, and remember that the thesis template has versions of Ecology and Science bsts which support webpage citation formats. 

\chapter{Tables and Graphics}

\section{Tables}
	The following section contains examples of tables, most of which have been commented out for brevity. (They will show up in the .tex document in red, but not at all in the .pdf). For more help in constructing a table (or anything else in this document), please see the LaTeX pages on the CUS site. 


	\clearpage 
%% \clearpage ends the page, and also dumps out all floats. 
%% Floats are things like tables and figures.

If you want to make a table that is longer than a page, you will want to use the longtable environment. Uncomment the table below to see an example, or see our online documentation.

%% An example of a long table, with headers that repeat on each subsequent page: Results from the summers of 1998 and 1999 work at Reed College done by Grace Brannigan, Robert Holiday and Lien Ngo in 1998 and Kate Brown and Christina Inman in 1999.

	\begin{longtable}{||c|c|c|c||}
	 	\caption[Chromium Hexacarbonyl Data Collected in 1998--1999]{Chromium Hexacarbonyl Data Collected in 1998--1999}\\ \hline
	    	  \multicolumn{4}{||c||}{Chromium Hexacarbonyl} \\\hline
		   State & Laser wavelength & Buffer gas & Ratio of $\frac{\textrm{Intensity
at vapor pressure}}{\textrm{Intensity at 240 Torr}}$ \\ \hline
		  \endfirsthead
		\hline     State & Laser wavelength & Buffer gas & Ratio of
$\frac{\textrm{Intensity at vapor pressure}}{\textrm{Intensity at 240 Torr}}$\\
\hline
		    \endhead

	    $z^{7}P^{\circ}_{4}$ & 266 nm & Argon & 1.5 \\\hline
	    $z^{7}P^{\circ}_{2}$ & 355 nm & Argon & 0.57 \\\hline
	    $y^{7}P^{\circ}_{3}$ & 266 nm & Argon & 1 \\\hline
	    $y^{7}P^{\circ}_{3}$ & 355 nm & Argon & 0.14 \\\hline
	    $y^{7}P^{\circ}_{2}$ & 355 nm & Argon & 0.14 \\\hline
	    $z^{5}P^{\circ}_{3}$ & 266 nm & Argon & 1.2 \\\hline
	    $z^{5}P^{\circ}_{3}$ & 355 nm & Argon & 0.04 \\\hline
	    $z^{5}P^{\circ}_{3}$ & 355 nm & Helium & 0.02 \\\hline
	    $z^{5}P^{\circ}_{2}$ & 355 nm & Argon & 0.07 \\\hline
	    $z^{5}P^{\circ}_{1}$ & 355 nm & Argon & 0.05 \\\hline
	    $y^{5}P^{\circ}_{3}$ & 355 nm & Argon & 0.05, 0.4 \\\hline
	    $y^{5}P^{\circ}_{3}$ & 355 nm & Helium & 0.25 \\\hline
	    $z^{5}F^{\circ}_{4}$ & 266 nm & Argon & 1.4 \\\hline
	    $z^{5}F^{\circ}_{4}$ & 355 nm & Argon & 0.29 \\\hline
	    $z^{5}F^{\circ}_{4}$ & 355 nm & Helium & 1.02 \\\hline
	    $z^{5}D^{\circ}_{4}$ & 355 nm & Argon & 0.3 \\\hline
	    $z^{5}D^{\circ}_{4}$ & 355 nm & Helium & 0.65 \\\hline
	    $y^{5}H^{\circ}_{7}$ & 266 nm & Argon & 0.17 \\\hline
	    $y^{5}H^{\circ}_{7}$ & 355 nm & Argon & 0.13 \\\hline
	    $y^{5}H^{\circ}_{7}$ & 355 nm & Helium & 0.11 \\\hline
	    $a^{5}D_{3}$ & 266 nm & Argon & 0.71 \\\hline
	    $a^{5}D_{2}$ & 266 nm & Argon & 0.77 \\\hline
	    $a^{5}D_{2}$ & 355 nm & Argon & 0.63 \\\hline
	    $a^{3}D_{3}$ & 355 nm & Argon & 0.05 \\\hline
	    $a^{5}S_{2}$ & 266 nm & Argon & 2 \\\hline
	    $a^{5}S_{2}$ & 355 nm & Argon & 1.5 \\\hline
	    $a^{5}G_{6}$ & 355 nm & Argon & 0.91 \\\hline
	    $a^{3}G_{4}$ & 355 nm & Argon & 0.08 \\\hline
	    $e^{7}D_{5}$ & 355 nm & Helium & 3.5 \\\hline
	    $e^{7}D_{3}$ & 355 nm & Helium & 3 \\\hline
	    $f^{7}D_{5}$ & 355 nm & Helium & 0.25 \\\hline
	    $f^{7}D_{5}$ & 355 nm & Argon & 0.25 \\\hline
	    $f^{7}D_{4}$ & 355 nm & Argon & 0.2 \\\hline
	    $f^{7}D_{4}$ & 355 nm & Helium & 0.3 \\\hline
	    \multicolumn{4}{||c||}{Propyl-ACT} \\\hline
%	    State & Laser wavelength & Buffer gas & Ratio of $\frac{\textrm{Intensity
%at vapor pressure}}{\textrm{Intensity at 240 Torr}}$\\ \hline
	    $z^{7}P^{\circ}_{4}$ & 355 nm & Argon & 1.5 \\\hline
	    $z^{7}P^{\circ}_{3}$ & 355 nm & Argon & 1.5 \\\hline
	    $z^{7}P^{\circ}_{2}$ & 355 nm & Argon & 1.25 \\\hline
	    $z^{7}F^{\circ}_{5}$ & 355 nm & Argon & 2.85 \\\hline
	    $y^{7}P^{\circ}_{4}$ & 355 nm & Argon & 0.07 \\\hline
	    $y^{7}P^{\circ}_{3}$ & 355 nm & Argon & 0.06 \\\hline
	    $z^{5}P^{\circ}_{3}$ & 355 nm & Argon & 0.12 \\\hline
	    $z^{5}P^{\circ}_{2}$ & 355 nm & Argon & 0.13 \\\hline
	    $z^{5}P^{\circ}_{1}$ & 355 nm & Argon & 0.14 \\\hline
	    \multicolumn{4}{||c||}{Methyl-ACT} \\\hline
%	    State & Laser wavelength & Buffer gas & Ratio of $\frac{\textrm{Intensity
% at vapor pressure}}{\textrm{Intensity at 240 Torr}}$\\ \hline
	    $z^{7}P^{\circ}_{4}$ & 355 nm & Argon & 1.6, 2.5 \\\hline
	    $z^{7}P^{\circ}_{4}$ & 355 nm & Helium & 3 \\\hline
	    $z^{7}P^{\circ}_{4}$ & 266 nm & Argon & 1.33 \\\hline
	    $z^{7}P^{\circ}_{3}$ & 355 nm & Argon & 1.5 \\\hline
	    $z^{7}P^{\circ}_{2}$ & 355 nm & Argon & 1.25, 1.3 \\\hline
	    $z^{7}F^{\circ}_{5}$ & 355 nm & Argon & 3 \\\hline
	    $y^{7}P^{\circ}_{4}$ & 355 nm & Argon & 0.07, 0.08 \\\hline
	    $y^{7}P^{\circ}_{4}$ & 355 nm & Helium & 0.2 \\\hline
	    $y^{7}P^{\circ}_{3}$ & 266 nm & Argon & 1.22 \\\hline
	    $y^{7}P^{\circ}_{3}$ & 355 nm & Argon & 0.08 \\\hline
	    $y^{7}P^{\circ}_{2}$ & 355 nm & Argon & 0.1 \\\hline
	    $z^{5}P^{\circ}_{3}$ & 266 nm & Argon & 0.67 \\\hline
	    $z^{5}P^{\circ}_{3}$ & 355 nm & Argon & 0.08, 0.17 \\\hline
	    $z^{5}P^{\circ}_{3}$ & 355 nm & Helium & 0.12 \\\hline
	    $z^{5}P^{\circ}_{2}$ & 355 nm & Argon & 0.13 \\\hline
	    $z^{5}P^{\circ}_{1}$ & 355 nm & Argon & 0.09 \\\hline
	    $y^{5}H^{\circ}_{7}$ & 355 nm & Argon & 0.06, 0.05 \\\hline
	    $a^{5}D_{3}$ & 266 nm & Argon & 2.5 \\\hline
	    $a^{5}D_{2}$ & 266 nm & Argon & 1.9 \\\hline
	    $a^{5}D_{2}$ & 355 nm & Argon & 1.17 \\\hline
	    $a^{5}S_{2}$ & 266 nm & Argon & 2.3 \\\hline
	    $a^{5}S_{2}$ & 355 nm & Argon & 1.11 \\\hline
	    $a^{5}G_{6}$ & 355 nm & Argon & 1.6 \\\hline
	    $e^{7}D_{5}$ & 355 nm & Argon & 1 \\\hline

		\end{longtable}

   
   \section{Figures}
   
	If your thesis has a lot of figures, \LaTeX\ might behave better for you than that other word processor.  One thing that may be annoying is the way it handles ``floats'' like tables and figures. \LaTeX\ will try to find the best place to put your object based on the text around it and until you're really, truly done writing you should just leave it where it lies.   There are some optional arguments to the figure and table environments to specify where you want it to appear; see the comments in the first figure.

	If you need a graphic or tabular material to be part of the text, you can just put it inline. If you need it to appear in the list of figures or tables, it should be placed in the floating environment. 
	
	To get a figure from StatView, JMP, SPSS or other statistics program into a figure, you can print to pdf or save the image as a jpg or png. Precisely how you will do this depends on the program: you may need to copy-paste figures into Photoshop or other graphic program, then save in the appropriate format.
	
	Below we have put a few examples of figures. For more help using graphics and the float environment, see our online documentation.
	
	And this is how you add a figure with a graphic:
	\begin{figure}[h]
	% the options are h = here, t = top, b = bottom, p = page of figures.
	% you can add an exclamation mark to make it try harder, and multiple
	% options if you have an order of preference, e.g.
	% \begin{figure}[h!tbp]
	   
	       \centering
	    % DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
	    \includegraphics{subdivision}
	     \caption{A Figure}
	 \label{subd}
	\end{figure}

\clearpage %% starts a new page and stops trying to place floats such as tables and figures

\section{More Figure Stuff}
You can also scale and rotate figures.
 	\begin{figure}[h!]
	   
	       \centering
	    % DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
	    \includegraphics[scale=0.5,angle=180]{subdivision}
	    % if your figure shows up not where you want it, it may just be too big to fit. You can use the scale argument to shrink it, e.g. scale=0.85 is 85 percent of the original size. 
	     \caption{A Smaller Figure, Flipped Upside Down}
	 \label{subd2}
	\end{figure}

\section{Even More Figure Stuff}
With some clever work you can crop a figure, which is handy if (for instance) your EPS or PDF is a little graphic on a whole sheet of paper. The viewport arguments are the lower-left and upper-right coordinates for the area you want to crop.

 	\begin{figure}[h!]
	    	       \centering
	    % DO NOT ADD A FILENAME EXTENSION TO THE GRAPHIC FILE
	   \includegraphics[clip=true, viewport=.0in .0in 1in 1in]{subdivision}
	    \caption{A Cropped Figure}
	 \label{subd3}
	\end{figure}
	
      \subsection{Common Modifications}
      The following figure features the more popular changes thesis students want to their figures. This information is also on the web at \url{web.reed.edu/cis/help/latex/graphics.html}.
    %\renewcommand{\thefigure}{0.\arabic{figure}} 	% Renumbers the figure to the type 0.x
    %\addtocounter{figure}{4} 						% starts the figure numbering at 4
    \begin{figure}[htbp]
    \begin{center}
   \includegraphics[scale=0.5]{subdivision}
    \caption[Subdivision of arc segments]{\footnotesize{Subdivision of arc segments. You can see that $ p_3 = p_6^\prime$.}} %the special ToC caption is in square brackets. The \footnotesize makes the figure caption smaller
    \label{barplot}
    \end{center}
    \end{figure} 

\chapter*{Conclusion}
         \addcontentsline{toc}{chapter}{Conclusion}
	\chaptermark{Conclusion}
	\markboth{Conclusion}{Conclusion}
	\setcounter{chapter}{4}
	\setcounter{section}{0}
	
Here's a conclusion, demonstrating the use of all that manual incrementing and table of contents adding that has to happen if you use the starred form of the chapter command. The deal is, the chapter command in \LaTeX\ does a lot of things: it increments the chapter counter, it resets the section counter to zero, it puts the name of the chapter into the table of contents and the running headers, and probably some other stuff. 

So, if you remove all that stuff because you don't like it to say ``Chapter 4: Conclusion'', then you have to manually add all the things \LaTeX\ would normally do for you. Maybe someday we'll write a new chapter macro that doesn't add ``Chapter X'' to the beginning of every chapter title.

\section{More info}
And here's some other random info: the first paragraph after a chapter title or section head \emph{shouldn't be} indented, because indents are to tell the reader that you're starting a new paragraph. Since that's obvious after a chapter or section title, proper typesetting doesn't add an indent there. 


%If you feel it necessary to include an appendix, it goes here.
    \appendix
      \chapter{The First Appendix}
      \chapter{The Second Appendix, for Fun}


%This is where endnotes are supposed to go, if you have them.
%I have no idea how endnotes work with LaTeX.

  \backmatter % backmatter makes the index and bibliography appear properly in the t.o.c...

% if you're using bibtex, the next line forces every entry in the bibtex file to be included
% in your bibliography, regardless of whether or not you've cited it in the thesis.
    \nocite{*}

% Rename my bibliography to be called "Works Cited" and not "References" or ``Bibliography''
% \renewcommand{\bibname}{Works Cited}

%    \bibliographystyle{bsts/mla-good} % there are a variety of styles available; 
%  \bibliographystyle{plainnat}
% replace ``plainnat'' with the style of choice. You can refer to files in the bsts or APA 
% subfolder, e.g. 
 \bibliographystyle{APA/apa-good}  % or
 %\bibliography{thesis}
 \bibliography{Updated_Bib}
 % Comment the above two lines and uncomment the next line to use biblatex-chicago.
 %\printbibliography[heading=bibintoc]

% Finally, an index would go here... but it is also optional.
\end{document}
